spirv_cross/src/wrapper.cpp

```diff
@@ -268,6 +268,26 @@ extern "C"
             } while (0);)
     }
 
+    ScInternalResult sc_internal_compiler_get_active_buffer_ranges(const ScInternalCompilerBase *compiler, uint32_t id, ScBufferRange **active_buffer_ranges, size_t *size)
+    {
+        INTERNAL_RESULT(
+            do {
+                auto const &comp = *((spirv_cross::Compiler *)compiler);
+                auto const &sc_active_buffer_ranges = comp.get_active_buffer_ranges(id);
```

Comment:
Can an exception thrown when `id` is invalid? `check!` is used for exception handling on the Rust side